docs/en/relying-party-instance.rst

```diff
@@ -0,0 +1,307 @@
+.. include:: ../common/common_definitions.rst
+
+.. _relying-party-instance:
+
+Relying Party Instance
+=========================
+
+The Relying Party Instance (RPI) is a mobile or embedded application designed to request, receive, and process Digital Credentials from Wallet Instances in a trusted manner. Each RPI ensures the integrity, confidentiality and authenticity of Credential exchanges, enabling secure interactions between Users and Relying Parties.
+
+There are two primary types of Relying Party Instances, each serving different operational environments:
+
+- **Embedded Relying Party Instance**: an hardware/software solution running on a specialized device (e.g., turnstiles). Each instance corresponds to a specific installation of the application on an embedded device. Since the trust with the Embedded Relying Party Instance is established through the trust established with the Mobile Relying Party Provider, the Embedded Relying Party Instance can be considered a confidential client.
+- **Mobile Relying Party Instance**: a native application running on a mobile device (e.g., smartphone or tablet). Each instance corresponds to a specific installation of the application on a device. Since the trust with the Mobile Relying Party Instance is established through the trust established with the Mobile Relying Party Provider, the Mobile Relying Party Instance can be considered a confidential client.
+- **Web Relying Party Instance**: a remote application operated by the Relying Party. A Web Relying Party Instance operates as a confidential client, meaning it can securely store confidential configurations (such as its private cryptographic keys) on a remote server. In this context, the Relying Party is unprovisioned and the presentation request and validation is automatically handled by software.
+
+.. note::
+  Mobile and Embedded Relying Party Instances acting as public clients are not considered within these specifications.
+
+.. note::
+  
+  Unlike the Web Relying Party Instance, a Mobile and Embedded Relying Party Instance requires proper lifecycle management and special registration procedures managed through the Relying Party Backend.
+
+Further technical and operational details are discussed in the following sections.
+
+
+Mobile Relying Party Instance
+--------------------------------
+
+A Mobile Relying Party Instance's lifecycle includes four main states: **Installed**, **Operational**, **Verified**, and **Uninstalled**, supporting functionalities such as registration, Access Certificate reissuance, and revocation. 
```

Comment:
In my opinion the operational state is confused with that of the Wallet Instance, what do you think about using another name? For example unverified or expired?